packages/versioning/test/repository.spec.ts

```diff
@@ -153,4 +242,98 @@ describe('Versioning', function () {
     expect(subSchemaFixture).toHaveProperty('$id');
     expect(subSchemaFixture['$id']).toBe(instance.buildRefSchemaUrl('sub-schema.schema.json'));
   });
+
+  it('generateReadableSchema should should resolve external references', async () => {
+    const writeFileSpy = jest.spyOn(utils, 'writeFile').mockImplementationOnce(() => Promise.resolve(undefined));
+    const schemaFilePath = resolve(__dirname, 'test-schema.json');
+    const writeFilePath = resolve(__dirname, 'readable-schema.json');
+    await SchemaRepositoryVersion.generateReadableSchema({ schemaFilePath });
+    expect(writeFileSpy).toBeCalledTimes(1);
+    expect(writeFileSpy).toBeCalledWith(writeFilePath, readableSchema);
+    writeFileSpy.mockReset();
+  });
```

Comment:
do you make the assumption that this test is valid because no error is thrown ?
if that’s the case you could make it explicit.
I’m asking since i don’t see any check done on the content of the generated schema.

Reply:
The check is in `expect(writeFileSpy).toBeCalledWith(writeFilePath, readableSchema);`, `readableSchema` is the value that we expect to be written. I can make it more explicit by adding `expect(bundledSchema).toEqual(readableSchema);` where `bundledSchema` is the return value.